jherr/create-mf-app
# Resolve the missing tsconfig.json for the react application using typescript.

## Code Fix
[x] Ensure tsconfig.json file is available after creating the React App using typescrpt.
## Issue URL
https://github.com/jherr/create-mf-app/issues/21
##  Steps to test: 
1. Checkout the code and run `npm run start`, ensure you choose framework = react and Language = typescript
2. After the project is created, ensure that tsconfig.json file is available
